git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
@benlast I think that is totally enough for visibility, but not enough for tooling (like JIRA integration). Definitely great step forward.